test/rfqm_test.ts

```diff
@@ -1151,6 +1168,67 @@ describe(SUITE_NAME, () => {
             const submissions = await dbUtils.findRfqmTransactionSubmissionsByOrderHashAsync(orderHash);
             expect(submissions[0].status).to.eq(RfqmTransactionSubmissionStatus.SucceededConfirmed);
 
+            mockAxios.reset();
+        });
+        it('should sucessfully resolve when there is a retry after last look is accepted', async () => {
+            const mockAxios = new AxiosMockAdapter(axiosClient);
+            mockAxios.onPost(`${MARKET_MAKER_1}/submit`).replyOnce(HttpStatus.OK, mmResponse);
+            // write a corresponding quote entity to validate against
+            await connection.getRepository(RfqmQuoteEntity).insert(mockQuote);
+
+            await request(app)
+                .post(`${RFQM_PATH}/submit`)
+                .send({ type: RfqmTypes.MetaTransaction, metaTransaction, signature: VALID_SIGNATURE })
+                .set('0x-api-key', API_KEY)
+                .expect(HttpStatus.CREATED)
+                .expect('Content-Type', /json/);
```

Comment:
If the `lastLook` is already `true`, I think we can skip the mocking in this test

(can remove the mockAxios stuff in this test)

Reply:
👍